content/syllabuses/web-dev-boot.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Web development boot camp
+---
+ - {{% contentlink "topics/free-code-camp/javascript-data-structures-and-algorithms/basic-javascript" %}}
+ - {{% contentlink "topics/web-dev-self-learning" %}} - concentrate on the Javascript learning only
```

Comment:
Rather point to specific parts of fcc I think. We can point them at other stuff and say "these resources might be useful" but I think it'll be good to tell them what we really expect from them.

Also we can measure their progress in FCC I think - there is a "public profile" page that is supposed to show people's progress

Reply:
They are pointed to the basic Js section on line 4